src/nccl_ofi_topo.cpp

```diff
@@ -326,6 +374,13 @@ static int get_info_for_node(hwloc_obj_t node, struct fi_info *info_list, struct
 		return 0;
 	}
 
+	/*
+	 * Check if we want to skip nics connected to root port
+	 */
+	if (ofi_nccl_skip_rp_nics.get() == 1 && is_rp_nic(node)) {
```

Comment:
`ofi_nccl_skip_rp_nics.get() && is_rp_nic(node)`.  `.get()` on a boolean parameter returns a `bool`, so no need to compare against `1`.